project.clj

```diff
@@ -139,7 +139,9 @@
                                        :main-namespace "puppetlabs.puppetdb.core"
                                        :create-varlib true}
                                 :config-dir "ext/config/pe"}
-                  :version ~pe-pdb-version
+                  :plugins [[puppetlabs/lein-ezbake "0.2.8"
```

Comment:
So the original way @waynr proposed running the build was in this PR when it was first introduced: https://github.com/puppetlabs/puppetdb/pull/1354

So it was `lein with-profile ezbake,pe ezbake build` which merged the profiles. This meant that you didn't need to repeat yourself by providing a second `:plugins` part. Perhaps we just need to bump this in the ezbake profile only, if this is how we plan on running the build for PE.